components/layout/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -1,19 +1,11 @@
 import AccountCircle from '@mui/icons-material/AccountCircle'
-import {
-  Box,
-  Divider,
-  List,
-  ListItem,
-  ListItemText,
-  SwipeableDrawer,
-  Link as MuiLink,
-  Typography,
-} from '@mui/material'
+import { Box, SwipeableDrawer } from '@mui/material'
 import { useRouter } from 'next/router'
 import { useTranslation } from 'next-i18next'
 
 import { HeaderAction } from '@/components/common'
 import { CategoryNestedNavigation } from '@/components/layout'
+import { MyAccountTemplate } from '@/components/page-templates'
```

Comment:
We don't want to render the My Account Template page inside of the HamburgerMenu

Reply:
done